src/new_components/Styling/index.js

```diff
@@ -339,7 +339,7 @@ export const Button = styled(SmartButton)`
     box-shadow: ${props => props.boxShadow};
 
     &:hover {
-        background-color: ${props => props.colorHover || props.color};
```

Comment:
removed `props.color` because, when moving the mouse to the button the backgroun changes to the text color